Models/instrumentation/GTX330/GTX330.nas

```diff
@@ -23,6 +25,12 @@ var mode_texts = {
   5 : "ALT"
 };
 
+var display_colors = {
+  "off"      : [   0,   0,   0], # color when the display is off, e.g. the battery master switch is OFF (usually black)
+  "active"   : [   1,   1,   0], # color of an active pixel (yellow)
```

Comment:
@HHS81 What do you think of the colors? The real GTX330 is not 100% yellow, I think.